organisaatio-service/src/main/resources/db/migration-test/R__fixed_test_data.sql

```diff
@@ -10,12 +10,14 @@ begin
         data    AS (SELECT 'test_data' as paivittaja, checker.newoid as oid, nimi from checker),
         mkt     AS (INSERT INTO monikielinenteksti (id,version) select (select max(id)+1 from monikielinenteksti),0 from data RETURNING id),
         mktval  AS (INSERT INTO monikielinenteksti_values (id, value, key) SELECT mkt.id,data.nimi,kieli.id FROM mkt join data on 1=1 join (values('fi'),('sv'),('en'))as kieli(id) on 1=1),
-        org     AS (INSERT into organisaatio (oid,id,version, nimi_mkt, piilotettu, organisaatiopoistettu, paivityspvm, paivittaja, alkupvm, kotipaikka, maa, parentidpath,oppilaitostyyppi, oppilaitoskoodi, nimihaku)
-                    SELECT  data.oid, (select coalesce(max(id)+1,1) from organisaatio),0, mkt.id, false, false, CURRENT_TIMESTAMP, data.paivittaja, CURRENT_DATE, 'kunta_153', 'maatjavaltiot1_fin', parentidpath, oppilaitostyyppi, oppilaitoskoodi, createOrganisaatio.nimi FROM mkt join data on 1=1 RETURNING id, oid, organisaatio.parentidpath ),
+        vaka_tt AS (INSERT INTO varhaiskasvatuksen_toimipaikka_tiedot (id, version, toimintamuoto, kasvatusopillinen_jarjestelma, paikkojen_lukumaara)
+                    SELECT nextval('public.hibernate_sequence'), 0, vardatoimintamuoto, 'vardakasvatusopillinenjarjestelma_kj98', 0 FROM checker WHERE vardatoimintamuoto IS NOT NULL RETURNING id),
+        org     AS (INSERT into organisaatio (oid,id,version, nimi_mkt, piilotettu, organisaatiopoistettu, paivityspvm, paivittaja, alkupvm, kotipaikka, maa, parentidpath,oppilaitostyyppi, oppilaitoskoodi, nimihaku, yritysmuoto, varhaiskasvatuksen_toimipaikka_tiedot_id)
+                    SELECT  data.oid, (select coalesce(max(id)+1,1) from organisaatio),0, mkt.id, false, false, CURRENT_TIMESTAMP, data.paivittaja, CURRENT_DATE, 'kunta_153', 'maatjavaltiot1_fin', parentidpath, oppilaitostyyppi, oppilaitoskoodi, createOrganisaatio.nimi, yritysmuoto, vaka_tt.id FROM mkt LEFT JOIN vaka_tt ON true join data on 1=1 RETURNING id, oid, organisaatio.parentidpath),
         orgnim  AS (INSERT INTO organisaatio_nimi (id, version, alkupvm, organisaatio_id, nimi_mkt, paivittaja) SELECT (select coalesce(max(id)+1,1) from organisaatio_nimi),0, CURRENT_DATE, org.id, mkt.id, data.paivittaja  from org join mkt on 1=1 join data on 1=1 returning id),
         orgtyp  AS (INSERT INTO organisaatio_tyypit (organisaatio_id, tyypit) SELECT org.id, typ.id from org join unnest(types) as typ(id) on 1=1 returning organisaatio_tyypit.organisaatio_id, organisaatio_tyypit.tyypit),
         kieli1  AS (INSERT INTO organisaatio_kielet (organisaatio_id, kielet) SELECT id, 'oppilaitoksenopetuskieli_1' from org),
-        parent  AS (INSERT INTO organisaatio_parent_oids (organisaatio_id, parent_position, parent_oid) SELECT org.id, 0, parentoid from org),
+        parent  AS (INSERT INTO organisaatio_parent_oids (organisaatio_id, parent_position, parent_oid) SELECT org.id, 0, parentoid from org where parentoid is not null),
```

Comment:
Lisäsin tän where ehdon, koska osa organisaatioista ei vaan suostunut tulla näkymään, jos sen juuriorganisaation `parentoid` oli sama kuin sen `oid`. Hakkasin päätä tähän vähän turhan pitkään enkä vieläkään tiedä mikä juuriorganisaation `parentoid` pitäis oikeasti olla. En ymmärrä, miten nää on aiemmin toiminut. Ehkä ei ole?